doc/dev/developer_guide/dash-devel.rst

```diff
@@ -215,8 +215,8 @@ The build process is based on `Node.js <https://nodejs.org/>`_ and requires the
 Prerequisites
 ~~~~~~~~~~~~~
 
- * Node 10.0.0 or higher
- * NPM 5.7.0 or higher
+ * Node 12.14.0 or higher
```

Comment:
Shouldn't this be aligned with the real version we're using?
https://github.com/ceph/ceph/blob/master/make-dist#L135

Reply:
You are right, I'll make the change